tools/topology/topology2/development/tplg-targets-bench.cmake

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+#
+# Append test topologies for multiple platforms and multiple formats
+# to TPLGS array.
+#
+
+set(platforms "TGL" "MTL")
+set(sampleformats "16" "24" "32")
+
+set(components
+	"dcblock"
+	"drc"
+	"drc_multiband"
+	"eqiir"
+	"eqfir"
+	"gain"
+)
+
+set(component_parameters
+	"BENCH_DCBLOCK_PARAMS=default"
+	"BENCH_DRC_PARAMS=enabled"
+	"BENCH_DRC_MULTIBAND_PARAMS=default"
+	"BENCH_EQIIR_PARAMS=highpass_50hz_0db_48khz"
+	"BENCH_EQFIR_PARAMS=loudness"
+	"BENCH_GAIN_PARAMS=default"
+)
+
+set(components_s32
+	"aria"
+)
+
+set(component_parameters_s32
+	"BENCH_ARIA_PARAMS=default"
+)
+
+foreach(p ${platforms})
+	string(TOLOWER ${p} pl)
+
+	# Add components with all sample formats
+	foreach(sf ${sampleformats})
+		foreach(comp bench_param IN ZIP_LISTS components component_parameters)
```

Comment:
Suggestion by  @marc-hb , need to bump tools/CMakeLists.txt:cmake_minimum_required(VERSION 3.13) to 3.17 for ZIP_LISTS.

Reply:
I would have to be `tools/CMakeLists.txt` unfortunately. Right now we have a top-level "tools" CMake project that makes no sense because it's a collection of totally unrelated tools that unfortunately share CMake things like `cmake_minimum_required()` for no good reason.

CMake 3.17 is 2 years old so I think that's not very demanding and acceptable.